guides/doc-Release_Notes/topics/foreman.adoc

```diff
@@ -4,8 +4,17 @@
 [id="foreman-headline-features"]
 == Headline Features
 
+=== Removed installer duplicate parameters
+
+Both `--foreman-logging-layout` and `--puppet-server-foreman-url` are no longer needed.
```

Comment:
https://github.com/theforeman/foreman-documentation/pull/2432 & https://github.com/theforeman/foreman-documentation/pull/2433 apply this to the docs here.